src/pseudopeople/interface.py

```diff
@@ -107,7 +107,7 @@ def _generate_dataset(
         noised_data = _extract_columns(dataset.columns, noised_data)
         noised_dataset.append(noised_data)
 
-    noised_dataset = pd.concat(noised_dataset, ignore_index=True)
+    noised_dataset = pd.concat(noised_dataset)
```

Comment:
Why do we need to do this?

Reply:
I needed indexes to match for the pytests that load the split/multiple sample datasets. This was an easy way to do that and should be safe since the index in the noised datasets don't actually mean anything.

Reply:
I don't follow. Which tests are these?